"fatal error: openssl/ssl.h: No such file or directory"

During installation I got the following error:
```
ERROR: Service 'php7console' failed to build: The command '/bin/sh -c docker-php-ext-install opcache calendar dba pcntl bcmath mbstring xmlrpc ftp shmop' returned a non-zero code: 2
```
I see a stacktrace above, here is the error:
```
/usr/src/php/ext/ftp/php_ftp.c:33:26: fatal error: openssl/ssl.h: No such file or directory
```

--- a/conf/php7console/Dockerfile
+++ b/conf/php7console/Dockerfile
@@ -59,6 +59,7 @@ RUN apt-get install -y zlib1g-dev \
     && docker-php-ext-install zip
 
 # common
+RUN apt-get install -y libssl-dev
 RUN docker-php-ext-install opcache calendar dba pcntl bcmath mbstring xmlrpc ftp shmop
 
 # preconf enviroment

--- a/conf/php7xdebug/Dockerfile
+++ b/conf/php7xdebug/Dockerfile
@@ -61,6 +61,7 @@ RUN pecl install xdebug \
     && echo "xdebug.remote_connect_back=0" >> /usr/local/etc/php/conf.d/xdebug.ini
 
 # common
+RUN apt-get install -y libssl-dev
 RUN docker-php-ext-install opcache calendar dba pcntl bcmath mbstring xmlrpc ftp shmop mysqli
 
 

